### PR TITLE
Create outline for NYC city file

### LINKED ISSUE #101: user kmmtechnology submitted this issue
- create a branch
- add commits on my own branch
- open a pull request
- collaborate and have a conversation
- make some more commits
- merge our branch into master

### PR DESCRIPTION
**Pull request by kmmtechnology**
this pull request adds a folder and file for NYC city

@halfaro1  what food do u like?
this pull request resolves #101

### CODE REVIEW

File: kmmtechnology/kmmtechnology-nyc.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# NYC
+
+### Restaurants
+
```
Comment (marinafurman) [inline kmmtechnology/kmmtechnology-nyc.md:6]: Practicing Comment :)

File: kmmtechnology/kmmtechnology-nyc.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# NYC
+
+### Restaurants
```
Comment (dawnvoh) [inline kmmtechnology/kmmtechnology-nyc.md:3]: @kmmtechnology, what's your favorite bakery?

File: kmmtechnology/kmmtechnology-nyc.md (hunk 1 above)
Comment (VarunVats9) [inline kmmtechnology/kmmtechnology-nyc.md:6]: Fake Comment :+1:

File: kmmtechnology/kmmtechnology-nyc.md (hunk 2 above)
Comment (VarunVats9) [inline kmmtechnology/kmmtechnology-nyc.md:3]: @kmmtechnology Way to go !

Comment (kmmtechnology) [inline kmmtechnology/kmmtechnology-nyc.md:3]: I do not have any personal preference but I do like mcdonald's food

**Pull request merged**